tests/test_isg.py

```diff
@@ -242,7 +242,7 @@ def __setup_geom():
     ig.isovol_meshsets[iv1]['bounds'] = (0., 5.)
     ig.isovol_meshsets[iv2] = {}
     ig.isovol_meshsets[iv2]['bounds'] = (5., 10.)
-    ig.levels = [5.]
+    ig.levels = [5., 100.]
```

Comment:
Should this `100` match the `10` above?  It might not matter for the test...?

Maybe these magic numbers should be replaced by variables.

Reply:
(I am not sure where "above" is, but I will explain this separately).

In PR #43 I introduced adding the value `arbmax` into the list of level values. `arbmax` is an arbitrary value just chosen to be greater than the maximum data value in the mesh. It's necessary to capture all space in an isovolume that is greater than the selected level value. In this case the test mesh max value is either 40 or 50 (I don't remember exactly without looking), so 100 is just an arbitrary max value that has to be added to the level list. In a normal run where the user would do the first step (`ivdb.py`) and supply levels that should be within the range of min and max data present. In the `ivdb.py` first step, the code adds this `arbmax` value as another level. Then the user/code goes to the second step (`isg.py`) and the `arbmax` value is already present in the levels list because it was added there by `ivdb.py`. Since this tests are just checking the second step `isg.py` this `__setup_geom()` function creates a usable second step object without having to go through the first step. So in this case, that `arbmax` value has to manually added. And since it's arbitrary, 100 is a fine value (where as 10 wouldn't work).

I can add a comment to explain that in the code.

Reply:
What might be more consistent is to add the value `45` instead of `100` since that's the `arbmax` I chose to add in line 16 for pr #43

Reply:
mkay- I was sloppy with this test, it should be a value of 10. I didn't realize I was using a completely different geometry here.